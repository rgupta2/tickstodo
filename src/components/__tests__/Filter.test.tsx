
import * as React from "react";
import * as renderer from "react-test-renderer";
import { Filter } from "../Filter";

it("FilterNavigation renders correctly", () => {
    const filter = "X";
    const currentFilter = "Y";
    const tree = renderer.create(
        <Filter
            filter={filter}
            currentFilter={currentFilter}
            onLinkClick={jest.fn}>
            {filter}</Filter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
