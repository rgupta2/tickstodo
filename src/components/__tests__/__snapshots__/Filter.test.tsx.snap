// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterNavigation renders correctly 1`] = `
<a
  href="#"
  onClick={[Function]}
>
  X
</a>
`;
